datumaro/plugins/cvat_format/converter.py

```diff
@@ -449,7 +552,9 @@ def apply(self):
         for subset_name, subset in self._extractor.subsets().items():
             ann_path = osp.join(self._save_dir, "%s.xml" % subset_name)
             with open(ann_path, "w", encoding="utf-8") as f:
-                writer = _SubsetWriter(f, subset_name, subset, self)
+                writer = _SubsetWriter(
+                    f, subset_name, subset, self, self._use_track, self._original_size
```

Comment:
If we already have an argument `context=self`, then I think it would be not necessary to add new arguments for `self._use_track` and `self._original_size`.